### PR TITLE
Basic startup for week-02-server and fix jetty version on this branch

### PR DESCRIPTION
**Pull request by SamRod33**
index.html
- Create hyperlink with message to give client "hello world!" response

RandomResponseServlet.java
- Create new java servlet that will be where I give clients random responses
- Create randString() that generates a random message every response
- Create randMessages string array to store all random messages
- Fix formatting (specifically 80 column limit)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    };
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) 
+    throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("<h3>" + randString() + "</h3>");
+    }
+
+    /** Returns a random message from [] */
```
Comment (GoogleyGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java:50]: What is [] ? I am assuming you mean the randMessages. Try to say things like this expliccitly.

File: portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+/**
+ * Handles requests sent to the /hello URL. Try running a server and navigating
+ * to /hello!
+ */
+@WebServlet("/rand_response")
+public class RandomResponseServlet extends HttpServlet {
+
+    /** Random set of messages gathered from google and 
+     * https://bestlifeonline.com/world-facts/. */
+    private String[] randMessages =
```
Comment (GoogleyGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java:18]: If this is not going to change, consider making it immutable with the final keyword.

**Review by GoogleyGabriel — APPROVED** [review]
Good style and good code, keep up the good work!

File: portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java [hunk 3]
```diff
@@ -6,13 +6,50 @@
...
+    };
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) 
+    throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("<h3>" + randString() + "</h3>");
+    }
+
+    /** Returns a random message from [] */
```
Comment (SofiaPSU) [inline portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java:50]: To comment out a single line you can also use //

**Review by SofiaPSU — APPROVED** [review]
I did not see anything else that needs to be changed other than what Gabriel already said. Good job on this commit! I liked the random facts!

File: portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java (hunk 1 above)
Comment (SamRod33) [reply portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java:50]: Yeah I must've been on autopilot making that javadoc. Thanks for the catch!

Comment (GoogleyGabriel) [reply portfolio/src/main/java/com/google/sps/servlets/RandomResponseServlet.java:50]: NP :) Also, I see you merged -- but I do not see the changes. Did you make the changes elsewhere? If not, I recommend rolling back the merge and applying the changes.

**Pull request merged**